src/ngrams/handler.py

```diff
@@ -0,0 +1,167 @@
+import json
+import re
+import string
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem.snowball import FrenchStemmer, SpanishStemmer
+from nltk.util import ngrams
+from collections import Counter, OrderedDict
+
+from textblob import TextBlob
+
+"""
+Sample request body payload
+{
+    "entries": ["entry1", "entry2"],
+    "ngrams": {
+        "unigrams": true,
+        "bigrams": true,
+        "trigrams": true
+    },
+    "enable_stopwords": false,
+    "enable_stemming": true,
+    "enable_case_sensitive": false
+    "max_ngrams_items": 10
+}
+"""
+
+MAX_NGRAMS_ITEMS = 10
+
+stemmer_en = nltk.stem.PorterStemmer()
+stemmer_fr = FrenchStemmer()
+stemmer_es = SpanishStemmer()
+
+stopwords_en = stopwords.words('english')
+stopwords_fr = stopwords.words('french')
+stopwords_es = stopwords.words('spanish')
```

Comment:
Why are we providing stopwords besides these 3 languages?

Reply:
@k9845 The entries primarily are from these three languages (as discussed with Jean)